src/main/java/com/algorand/algosdk/algod/client/JSON.java

```diff
@@ -32,6 +32,7 @@
 import java.util.Date;
 import java.util.Map;
 
+@Deprecated
 public class JSON {
```

Comment:
I noticed there are some public static class declaration inside `JSON`, do we want to mark them as deprecated?

Reply:
I think externally, it should still be marked as deprecated (e.g. if you do something like `JSON.static_class()`, the annotation will appear on the first `JSON`). Admittedly, I only tested this on the latest `javac` version.

Reply:
got it, then I resolve it.